releases/0.9.0-incubating/INSTALL.md

```diff
@@ -36,60 +36,65 @@ You need to install gpg on your local machine.
 For Ubuntu user, run the following command:
 
 ```
-$ sudo apt-get install gnupg
+sudo apt-get install gnupg
 ```
 
 For Mac user, run the following command:
 
 ```
-$ brew install gpg
+brew install gpg
 ```
 
 The public key used to verify the checksums can be found [here](https://dist.apache.org/repos/dist/dev/incubator/openwhisk/KEYS). After
 download it, you need to import it on your local machine. 
 
 ```
-$ gpg --import <key_file>
+gpg --import <key_file>
 ```
 
 The parameter <key_file> is the file, where the public key is saved.
 
 To generate the SHA512 checksum:
 
 ```
-$ gpg --print-md SHA512 <artifact>
+gpg --print-md SHA512 <artifact>
 ```
 
 The parameter <artifact> is the file of the artifact "openwhisk-0.9.0-incubating-sources.tar.gz". Compare the content with the [SHA512 file](https://dist.apache.org/repos/dist/dev/incubator/openwhisk/apache-openwhisk-0.9.0-incubating-rc1/openwhisk-0.9.0-incubating-sources.tar.gz.sha512).
 
 Download the [signature](https://dist.apache.org/repos/dist/dev/incubator/openwhisk/apache-openwhisk-0.9.0-incubating-rc1/openwhisk-0.9.0-incubating-sources.tar.gz.asc), and verify it with the command:
 
 ```
-$ gpg --verify openwhisk-0.9.0-incubating-sources.tar.gz.asc openwhisk-0.9.0-incubating-sources.tar.gz
+gpg --verify openwhisk-0.9.0-incubating-sources.tar.gz.asc openwhisk-0.9.0-incubating-sources.tar.gz
 ```
 
 
 # Installation of OpenWhisk 0.9.0
 
 This instruction walks you through the steps to install OpenWhisk 0.9.0. We support both Ubuntu and Mac operating systems.
-Please download the source code package of OpenWhisk "openwhisk-0.9.0-incubating-sources.tar.gz" and unzip it, then you
-get a directory called "incubator-openwhisk-<version>" on your local machine.
+Please download the source code package of OpenWhisk "openwhisk-0.9.0-incubating-sources.tar.gz", unzip and extract it, then
+you get a directory called "incubator-openwhisk-<version>" on your local machine.
 
+You can use the following command to untar the package:
+
+```
+tar -xvzf openwhisk-0.9.0-incubating-sources.tar.gz 
+```
 
 ## Prerequisites
 
 If you are a Ubuntu user, our suggested version is between 14.04 and 16.04. Open a terminal, go to the directory of "incubator-openwhisk",
 and run the script "all.sh" under tools/ubuntu-setup:
 
 ```
-$ cd incubator-openwhisk-<version>
-$ ./tools/ubuntu-setup/all.sh
+cd incubator-openwhisk-<version>;
```

Comment:
Remove ;

Reply:
Is there version name not known to make the cd command complete?

Reply:
I suggest moving OPENWHISK_HOME paragraph here and using the property for the cd command as well.

Reply:
done.